### PR TITLE
Support table markers

### PR DESCRIPTION
**Pull request by BryanHo10**

### CODE REVIEW

File: USFMToolsSharp.Renderers.Docx/DocxRenderer.cs [hunk 1]
```diff
@@ -182,6 +181,29 @@ private void RenderMarker(Marker input, StyleConfig styles, XWPFParagraph parent
                         RenderMarker(marker, markerStyle, parentParagraph);
                     }
                     break;
+
+            // Table Markers
+                case TableBlock table:
+                    XWPFTable tableContainer = newDoc.CreateTable();
+
+                    // Clear Borders
+                    tableContainer.SetBottomBorder(XWPFTable.XWPFBorderType.NONE,0,0,"#FFFFFFF");
```
Comment (rbnswartz) [inline USFMToolsSharp.Renderers.Docx/DocxRenderer.cs]: is the color of the border needed if you're hiding it?

File: USFMToolsSharp.Renderers.Docx/DocxRenderer.cs [hunk 2]
```diff
@@ -281,7 +303,51 @@ public void createBookHeaders(string bookname)
...
+            switch (input)
+            {
+                case THMarker tHMarker:
+                    markerStyle.isBold = true;
+                    foreach (Marker marker in input.Contents)
+                    {
+                        RenderMarker(marker, markerStyle, cellContents);
+                    }
+                    break;
+                case THRMarker tHRMarker:
```
Comment (rbnswartz) [inline USFMToolsSharp.Renderers.Docx/DocxRenderer.cs]: I'm fairly certain THR is right aligned.

File: USFMToolsSharp.Renderers.Docx/DocxRenderer.cs [hunk 3]
```diff
@@ -281,7 +303,51 @@ public void createBookHeaders(string bookname)
...
+                        RenderMarker(marker, markerStyle, cellContents);
+                    }
+                    break;
+                case TCMarker tCMarker:
+                    foreach (Marker marker in input.Contents)
+                    {
+                        RenderMarker(marker, markerStyle, cellContents);
+                    }
+                    break;
+                case TCRMarker tCRMarker:
```
Comment (rbnswartz) [inline USFMToolsSharp.Renderers.Docx/DocxRenderer.cs]: same with TCR

File: USFMToolsSharp.Renderers.Docx/DocxRenderer.cs [hunk 4]
```diff
@@ -281,7 +303,51 @@ public void createBookHeaders(string bookname)
 
             bookNameCount++;
         }
-
+        public void getRenderedRows(Marker input, StyleConfig config,XWPFTable parentTable)
+        {
+            XWPFTableRow tableRowContainer = parentTable.CreateRow();
+            foreach (Marker marker in input.Contents)
```
Comment (rbnswartz) [inline USFMToolsSharp.Renderers.Docx/DocxRenderer.cs]: I'm fairly certain the cells need to be rendered in they're specified order but don't worry about that for the time being.

File: USFMToolsSharp.Renderers.Docx/DocxRenderer.cs (hunk 1 above)
Comment (BryanHo10) [reply USFMToolsSharp.Renderers.Docx/DocxRenderer.cs]: Tables are preset with black borders and I didn't find a method to set no Borders.

File: USFMToolsSharp.Renderers.Docx/DocxRenderer.cs (hunk 2 above)
Comment (BryanHo10) [reply USFMToolsSharp.Renderers.Docx/DocxRenderer.cs]: @rbnswartz Updated

File: USFMToolsSharp.Renderers.Docx/DocxRenderer.cs (hunk 1 above)
Comment (rbnswartz) [reply USFMToolsSharp.Renderers.Docx/DocxRenderer.cs]: I mean you're setting `XWPFTable.XWPFBorderType.NONE` so I would guess that sets it to none?

Comment (BryanHo10) [reply USFMToolsSharp.Renderers.Docx/DocxRenderer.cs]: @rbnswartz there's no overloads for this function...I have to fill out each param. There's also no setter for BorderType.

File: USFMToolsSharp.Renderers.Docx/DocxRenderer.cs (hunk 4 above)
Comment (BryanHo10) [reply USFMToolsSharp.Renderers.Docx/DocxRenderer.cs]: @rbnswartz Are you referring to the Column position?

File: USFMToolsSharp.Renderers.Docx/DocxRenderer.cs (hunk 1 above)
Comment (rbnswartz) [reply USFMToolsSharp.Renderers.Docx/DocxRenderer.cs]: Ok you needed to add the color. Ok that makes sense

**Pull request merged**